### PR TITLE
fix: 181651455 typos in newsletter templates

### PR DESCRIPTION
**Pull request by eugbyte**
fix typos for news letter

### CODE REVIEW

File: maizzle/src/templates/newsletterVerify.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Notarise: Please confirm your email"
+title: "Notarise.gov.sg: Please confirm your email address"
```
Comment (zawmyolatt) [inline maizzle/src/templates/newsletterVerify.html]: keep the title same with other template like `Notarise:`

**Comment by zawmyolatt** [comment]
change pr title as `fix: issue title`

File: maizzle/src/templates/newsletterVerify.html (hunk 1 above)
Comment (eugbyte) [reply maizzle/src/templates/newsletterVerify.html]: Hmm ok let me clarify with the UX

Comment (zawmyolatt) [reply maizzle/src/templates/newsletterVerify.html]: i don't think it is use in somewhere. it's just template title for maizzle right ?

Comment (eugbyte) [reply maizzle/src/templates/newsletterVerify.html]: yup, just for the news letter emails

**Review by zawmyolatt — APPROVED** [review]

**Pull request merged**